### PR TITLE
Configure the Cucumber runner and first steps

### PR DESCRIPTION
**Pull request by Lc-Ana**

### CODE REVIEW

File: src/main/java/stepdefinitions/LoginSteps.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        accountPage.showErrorMessageWithBadCredentials();
+    }
+
+    @When("^user clicks on logout button from navigation bar$")
+    public void userClicksOnLogoutButtonFromNavigationBar() {
+        accountPage.navigateToLogoutPage();
+    }
+
+    @Given("^user is logged in open cart webpage$")
+    public void userIsLoggedInOpenCartWebpage() throws InterruptedException {
```
Comment (AndreiSecu1997) [inline src/main/java/stepdefinitions/LoginSteps.java:51]: throws exception, sure?

File: src/main/java/utils/TestContext.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package utils;
+
+import managers.WebDriverManager;
+
+public class TestContext {
+    private WebDriverManager webDriverManager;
+
+    public TestContext(){
+        webDriverManager = new WebDriverManager("chrome");
```
Comment (AndreiSecu1997) [inline src/main/java/utils/TestContext.java:9]: this parameter should be read from properties file

**Comment by AndreiSecu1997** [comment]
M-ai impresionat cu Login Feature. Large PR but good one. Congrats.

File: src/main/java/stepdefinitions/LoginSteps.java (hunk 1 above)
Comment (Lc-Ana) [reply src/main/java/stepdefinitions/LoginSteps.java:51]: There was a problem, initially I used Thread.sleep to understand what is happening, then resolved and I forgot to delete the exeption

> throws exception, sure?

**Pull request merged**